docs/tutorial/getting-started-with-istio-mesh.md

```diff
@@ -0,0 +1,259 @@
+
+### Step 2: Offer Istio Ingress
+
+Make the `istio-ingress-k8s` charm available to other Juju models so applications from other Juju models can be ingressed using the `istio-ingress-k8s` charm by [offering](https://documentation.ubuntu.com/juju/3.6/reference/juju-cli/list-of-juju-cli-commands/offer/) it:
+
+```bash
+juju offer istio-ingress-k8s:ingress-unauthenticated
+```
+
+
+## Deploy Charmed Bookinfo Application
+
+The Bookinfo application consists of three charms:
+- **`bookinfo-productpage-k8s`**: Frontend charm that displays book information
+- **`bookinfo-details-k8s`**: Backend charm that provides book details
+- **`bookinfo-reviews-k8s`**: Backend charm that provides book reviews 
+
+The Bookinfo example was chosen for this tutorial because:
+- It's the official demo application used by Istio
+- It's composed of multiple microservices that replicate a conventional web application and can be conveniently used to demonstrate various service mesh features
+
+The Bookinfo application needs the `bookinfo-productpage-k8s` successfully communicating with the `bookinfo-details-k8s` and `bookinfo-reviews-k8s` for the web application to display all the relevant information without any errors.
+
+![Charmed bookinfo application architecture](../assets/images/charmed-bookinfo-architecture.drawio.png)
+
+### Step 3: Deploy Application Components
+
+Create a model for the application and deploy the charms:
+
+```bash
+juju add-model bookinfo
+
+# Deploy the charms
+juju deploy bookinfo-productpage-k8s 
+juju deploy bookinfo-details-k8s
+juju deploy bookinfo-reviews-k8s
+
+# Connect the charms
+juju integrate bookinfo-productpage-k8s:details bookinfo-details-k8s:details
+juju integrate bookinfo-productpage-k8s:reviews bookinfo-reviews-k8s:reviews
+
+```
+
+Now you can sit back and watch the deployment take place:
+
+```bash
+juju status --integrations --watch=5s
+```
+
+### Step 4: Configure External Access
+
+Expose the `bookinfo-productpage-k8s` using the `istio-ingress-k8s` ingress gateway. It is necessary to [consume](https://documentation.ubuntu.com/juju/3.6/reference/juju-cli/list-of-juju-cli-commands/consume/) `istio-ingress-k8s` from the `istio-system` model into the `bookinfo` model before adding the ingress relation.
+
+```bash
+juju consume istio-system.istio-ingress-k8s
+juju integrate bookinfo-productpage-k8s istio-ingress-k8s:ingress-unauthenticated
+```
+
+This ingress relation would allow the `bookinfo-productpage-k8s` charm to be accessed from outside the cluster, for example, your browser.
+
+### Step 5: Access the Application
+
+Get the application URL and verify it's working:
+
+```bash
+juju run bookinfo-productpage-k8s/0 get-url
+```
+
+Open the URL in your browser. You'll see the book information page.
+
+![Bookinfo application](../assets/images/bookinfo-details-reviews.png)
+
+## Why Do We Need Charmed Istio?
```

Comment:
This is an example of a good section we could move to its own page.  The value statement of service mesh is something people outside this guide would like to know too.

If we moved this elsewhere, we could here just say that we want Istio because it easily adds mTLS and authorization control, and leave the deeper explanation to the link